apps/widclkinfo/widget.js

```diff
@@ -47,4 +57,20 @@ if (!require("clock_info").loadCount) { // don't load if a clock_info was alread
       }
     }
   };
-}
\ No newline at end of file
+
+  Bangle.on("hidden", () => {
+    console.log("hidden");
+    clockInfoMenu.y = -24;
+    if (clockInfoMenu.focus) {
+      clockInfoMenu.force_blur();
```

Comment:
Because we've checked `focus` above, I think it's safe to just have the `blur` method and avoid the cost of an extra jsvar, wdyt?

Reply:
I don't think `clockInfoMenu.blur()` is exposed. I chose the name `force_blur` to avoid internal name confusion inside clock_info. Also, clock_info exposes focus as a boolean but internally uses focus as a function, so I wasn't sure how to keep names consistent

Reply:
Ah I see your problem - I think that's ok to have the internal focus and the exposed one, we could rename the internal one so we present an API that doesn't give away internal details if you like?